app/components/ListDivider/ListDivider.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import { View, StyleSheet } from 'react-native';
+import { Colors } from '../../theme/theme';
+
+export default class ListDivider extends Component {
+  render() {
+    const css = StyleSheet.create({
+      divider: {
```

Comment:
I guess so. All this file does is create a line with some padding around it. So, it seemed a little crazy to break up the css into a different file. This file itself is almost a little overkill, but I made it so that the code was more declarative rather than imperative.